.github/workflows/ci.yml

```diff
@@ -42,5 +42,16 @@ jobs:
       - name: Run tests with Node version ${{ matrix.node-version }}
         uses: docker/build-push-action@v2.3.0
         with:
+          tags: calendar-date
           build-args: |
             NODE_VERSION=${{ matrix.node-version }}
+      - uses: shrink/actions-docker-extract@v1
+        id: extract
+        with:
+          image: 'calendar-date'
```

Comment:
This won't work, as the coverage files won't be in the final image. You need to build with a target or run the tests in the container by passing a command. I'd rather try to extract directly from the container used for testing, as it should be available at some point during the workflow.